code/__DEFINES/gamemode.dm

```diff
@@ -32,7 +32,7 @@
 // antag bantypes macros.
 #define ROLE_BANTYPE_BORER ROLE_BORER
 #define ROLE_BANTYPE_MALFUNCTION ROLE_MALFUNCTION
-#define ROLE_BANTYPE_TRAITOR ROLE_TRAITOR
+#define ROLE_BANTYPE_CONTRACTOR ROLE_CONTRACTOR
```

Comment:
Changing this will probably remove all the traitor bans from everyone.

Reply:
__Good.__

Reply:
That will only Impact the Russian community so I don't see it as a problem.